packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -82,80 +68,58 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
         callbackFn,
         data,
         domain,
-        done,
-        doneReference: state('done'),
       })
 
-      const commandsManager = new CommandsManager({
-        communicator,
-        isDoneFnAvailable: !!done,
-        userInvocationStack,
-      })
-
-      const logsManager = new LogsManager({
-        communicator,
-        userInvocationStack,
-      })
+      return new Bluebird((resolve, reject) => {
+        const onError = (err) => {
+          log.error(err)
```

Comment:
Did we want to call `correctStackForCrossDomainError` here or save that for a follow up or #20256? Some of the returned errors are bit lengthy 😅 
![](https://user-images.githubusercontent.com/3980464/154753406-b0d0ed03-811c-4908-9ef2-9485b0c13d8b.png)

Reply:
Looks like that only happens if you leave the done in the test after removing the fail handler, which isn't really a valid test. It's a much cleaner error if you remove the done. I'm not sure why it isn't showing the code frame though...

Reply:
Makes sense. I can just picture a user running into something like this and having some issues parsing what went wrong initially. I think the codeFrame is missing since we aren't reprocessing the error coming back from the secondary, but not 100% sure.